stagemonitor-core/src/main/java/org/stagemonitor/core/util/HttpClient.java

```diff
@@ -182,4 +181,14 @@ public Integer handleResponse(InputStream is, Integer statusCode, IOException e)
 			return statusCode;
 		}
 	}
+
+  	private String removeUserInfo(String url) {
```

Comment:
I would have to make a separate static logger for this method then? 	  		
logger.warn("Suppressed exception", e);